spec/models/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post do
+  subject = Post.new(author: User.first, title: 'love', text: 'Hello word', comments_counter: 10, likes_counter: 10)
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should not be exceed 250 char' do
+    subject.title = 'a' * 251
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments_counter must be greater than or equal to zero' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments_counter must be greater than or equal to zero' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Same here, Please add tests for `recent_comments` and `update_post_counter` methods in the post model.
- [ ] You should test all models as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add tests for `like `and `comment `models too.